lib/absinthe/phoenix/application.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Absinthe.Phoenix.Application do
```

Comment:
I think this is probably only needed for running tests, since the pubsub of the application is reused?

Reply:
May I suggest using start_supervised in the phoenix_test?

```elixir
setup_all do 
  _pid = start_supervised!(Phoenix.PubSub.child_spec(name: Absinthe.Phoenix.PubSub))
end
```